### DISCUSSION TITLE
iOS Contribution Map Widget missing?

### DISCUSSION THREAD

**Discussion by NotAnmol** [community/community#3316] [Mobile] [answered]
I can't find the new Github's widget in the widget drawer. Is anyone also facing the same issue.

---

SOLVED: Apparently i just had to open the app

**Comment by eliperkins** [comment, marked as answer]
👋  You're not alone in finding some iOS glitches related to the widget! It seems like the best steps to addressing this are:

1. Launch the app, else
2. Delete the app and reinstall, else
3. [If you're on a TestFlight build, install the App Store version to work around an iOS bug](https://developer.apple.com/forums/thread/655393)

**Reply by D3V4N5H** [reply]
How and where do I thank the developers for the 1 X 1 widget? I'm feeling grateful.
